README.md

```diff
@@ -6,36 +6,37 @@
 [![DependentBot](https://api.dependabot.com/badges/status?host=github&repo=SAP/task-explorer)](https://dependabot.com/)
 [![REUSE status](https://api.reuse.software/badge/github.com/SAP/task-explorer)](https://api.reuse.software/info/github.com/SAP/task-explorer)
 
-# VSCode Task Explorer
+# VS Code Task Explorer
```

Comment:
Please change it to Task Explorer, because the name has changed

Reply:
done